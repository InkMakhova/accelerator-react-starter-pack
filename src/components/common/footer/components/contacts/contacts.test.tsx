import React from 'react';
import { render } from '@testing-library/react';
import Contacts from './contacts';

it('should render Contacts correctly', () => {
  const {container} = render(<Contacts />);
  expect(container).toMatchSnapshot();
});
