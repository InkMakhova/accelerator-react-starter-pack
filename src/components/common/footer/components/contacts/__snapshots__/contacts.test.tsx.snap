// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Contacts correctly 1`] = `
<div>
  <section
    class="footer__nav-section footer__nav-section--contacts"
  >
    <h2
      class="footer__nav-title"
    >
      Контакты
    </h2>
    <p
      class="footer__nav-content"
    >
      г. Санкт-Петербург,
      <br />
       м. Невский проспект, 
      <br />
      ул. Казанская 6.
    </p>
    <div
      class="footer__nav-content"
    >
      <svg
        aria-hidden="true"
        class="footer__icon"
        height="8"
        width="8"
      >
        <use
          xlink:href="#icon-phone"
        />
      </svg>
      <a
        class="link"
        href="tel:88125005050"
      >
         8-812-500-50-50
      </a>
    </div>
    <p
      class="footer__nav-content"
    >
      Режим работы:
      <br />
      <span
        class="footer__span"
      >
        <svg
          aria-hidden="true"
          class="footer__icon"
          height="13"
          width="13"
        >
          <use
            xlink:href="#icon-clock"
          />
        </svg>
        <span>
           с 11:00 до 20:00
        </span>
        <span>
          без выходных
        </span>
      </span>
    </p>
  </section>
</div>
`;
